playbooks/deploy_vms_standalone.yaml

```diff
@@ -29,10 +29,20 @@
         var: hostvars[item]
         verbosity: 2
       with_items: "{{ groups['VMs'] }}"
+    - name: Check tmp folder permission
+      file:
+        path: "{{ qcow2tmpuploadfolder }}"
+        state: directory
+        owner: "{{ ansible_user }}"
+        group: "{{ ansible_user }}"
+        mode: '0755'
+      when: qcow2tmpuploadfolder is defined
     - name: Copy the disk on target
       copy:
         src: "{{ hostvars[item].vm_disk }}"
         dest: "{{ disk_pool }}/{{ hostvars[item].inventory_hostname }}.qcow2"
+      vars:
+        ansible_remote_tmp: "{{ qcow2tmpuploadfolder | default(omit) }}"
```

Comment:
It seems to me that the variable `ansible_remote_tmp` should not be defined in the playbook, but either in the inventory or directly in ansible.cfg.
Otherwise, looks good to me.

Reply:
No, it is an exception here because qcow2 disk cannot fit inside tmp.